src/main/java/org/jabref/model/strings/StringUtil.java

```diff
@@ -758,7 +758,7 @@ public static String quoteStringIfSpaceIsContained(String string) {
      * @param s The string to check
      * @return {@code True} if the given string does contain at least one whitespace character, {@code False} otherwise
      * */
-    public static boolean containsAnyWhitespaceCharacters(String s) {
-        return s.matches(".*\s.*");
+    public static boolean containsWhitespace(String s) {
```

Comment:
You now need to adapt the failing architecture test to the new size of the StringUtil class.
PS: I wasn't aware of the String.chars method, that's neat

Reply:
Increased it to 765